Azure/azure-cli
# [AKS] az aks create: add --enable-aad argument

**Description**  

This PR is to portal the support of creating AKS-managed Azure AD cluster from aks-preview extension. It adds a new argument `--enable-aad` to `az aks create` command. Customers can specify the `--aad-admin-group-object-ids` and `--aad-tenant-id` when creating/updating AKS cluster.

**Testing Guide**  


* Create an AKS-managed Azure AD cluster

```
az aks create -g MyResourceGroup -n MyManagedCluster --enable-aad [--aad-admin-group-object-ids <id>] [--aad-tenant-id <id>]
```

* Update an AKS-managed Azure AD cluster

```
az aks update -g MyResourceGroup -n MyManagedCluster [--aad-admin-group-object-ids <id>] [--aad-tenant-id <id>]
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.
